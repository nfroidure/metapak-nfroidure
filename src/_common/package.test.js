'use strict';

const packageTransformer = require('./package');

describe('Package transformer', () => {
  it('should work with an empty package.json', () => {
    expect(packageTransformer({})).toMatchSnapshot();
  });
});
