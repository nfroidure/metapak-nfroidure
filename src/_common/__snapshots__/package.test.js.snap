// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Package transformer should work with an empty package.json 1`] = `
Object {
  "author": "Nicolas Froidure",
  "config": Object {
    "commitizen": Object {
      "path": "./node_modules/cz-conventional-changelog",
    },
  },
  "devDependencies": Object {
    "commitizen": "^2.10.1",
    "conventional-changelog-cli": "^2.0.5",
    "cz-conventional-changelog": "^2.1.0",
  },
  "engines": Object {
    "node": ">=6.9.5",
  },
  "greenkeeper": Object {
    "ignore": Array [
      "commitizen",
      "cz-conventional-changelog",
      "conventional-changelog-cli",
    ],
  },
  "license": "MIT",
  "scripts": Object {
    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s",
    "cli": "env NODE_ENV=\${NODE_ENV:-cli}",
    "cz": "env NODE_ENV=\${NODE_ENV:-cli} git cz",
    "lint": "echo \\"WARNING: No linter specified\\"",
    "preversion": "npm t && npm run lint && npm run metapak -s",
    "test": "echo \\"WARNING: No tests specified\\"",
    "version": "npm run changelog && git add CHANGELOG.md",
  },
  "version": "0.0.0",
}
`;
